cmd/kubeadm/app/phases/controlplane/manifests_test.go

```diff
@@ -615,7 +761,7 @@ func TestGetControllerManagerCommand(t *testing.T) {
 		sort.Strings(actual)
 		sort.Strings(rt.expected)
 		if !reflect.DeepEqual(actual, rt.expected) {
-			t.Errorf("failed getControllerManagerCommand:\nexpected:\n%v\nsaw:\n%v", rt.expected, actual)
+			errorDiffArguments(t, rt.name, actual, rt.expected)
```

Comment:
Thanks! Do this for the scheduler args as well to be consistent?

Reply:
yes, tested by breaking it

Reply:
Here is an example:
```
=== RUN   TestGetControllerManagerCommand                                                                                     
--- FAIL: TestGetControllerManagerCommand (0.00s)                                                                             
        manifests_test.go:647: [custom IPv6 networking] failed getAPIServerCommand:                                           
                expected:                                                                                                     
                [--address=127.0.0.1 --allocate-node-cidrs=true --cluster-cidr=2001:db8::/66 --cluster-signing-cert-file=/var/
lib/certs/ca.crt --cluster-signing-key-file=/var/lib/certs/ca.key --controllers=*,bootstrapsigner,tokencleaner --kubeconfig=/e
tc/kubernetes/controller-manager.conf --leader-elect=true --node-cidr-mask-size=80 --root-ca-file=/var/lib/certs/ca.crt --serv
ice-account-private-key-file=/var/lib/certs/sa.key --use-service-account-credentials=true kube-controller-manager]            
                saw:                                                                                                          
                [--address=127.0.0.1 --allocate-node-cidrs=true --cluster-cidr=2001:db8::/64 --cluster-signing-cert-file=/var/
lib/certs/ca.crt --cluster-signing-key-file=/var/lib/certs/ca.key --controllers=*,bootstrapsigner,tokencleaner --kubeconfig=/e
tc/kubernetes/controller-manager.conf --leader-elect=true --node-cidr-mask-size=80 --root-ca-file=/var/lib/certs/ca.crt --serv
ice-account-private-key-file=/var/lib/certs/sa.key --use-service-account-credentials=true kube-controller-manager]            
                expectedShort:                                                                                                
                [--cluster-cidr=2001:db8::/66]                                                                                
                sawShort:                                                                                                     
                [--cluster-cidr=2001:db8::/64]                                                                                
                                                                
```